client/src/components/HtModal/HtModal.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { ReactElement } from 'react';
+import {
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+  ModalHeaderProps,
+  ModalProps,
+  ModalTitle,
+} from 'react-bootstrap';
+
+interface HtModalProps extends ModalProps {
+  showModal: boolean;
+  handleClose: () => void;
+  className?: string;
```

Comment:
There's a couple prop keys we should be able to get rid of since we're extending ModalProps like `className`, `closeButton` and `children` which are already included in the [react-bootstrap api](https://react-bootstrap.github.io/components/modal/#api)

After that, we could remove `HtModalHeaderProps` and just pass react-bootstraps native `ModalHeaderProps` to our new `HtModal.Header`. Or we could leave it as a placeholder so we already have a place to extend `ModalHeaderProps` in the future. @Badrri-Narayanan What do you think? I'm partial to removing unecessary stuff but you're you're more familar with the new component.

Reply:
@dpgraham4401 thanks for review comments. I will revisit and remove any redundant code after creating an issue.